src/main/java/com/toptrumps/core/DeckParser.java

```diff
@@ -7,7 +7,7 @@
 public class DeckParser {
 
     private ArrayList<Card> cards;
-    private final String FILE_PATH = "../src/main/resources/assets/StarCitizenDeck.txt";
+    private final String FILE_PATH = "src/main/resources/assets/StarCitizenDeck.txt";
```

Comment:
This path doesn’t work for me when I try to compile and run. Can you remind me to look at this with you? (This isn’t something that needs changing!)